### PR TITLE
i18n(pt-BR): update `guides/typescript.mdx`

### PR DESCRIPTION
**Pull request by viniciusdeliz**
- Changes to match main original file commit 9d4f1c6

#### Description (required)

Update `pt-br/guides/typescript.mdx` to match `en/guides/typescript.mdx`

### CODE REVIEW

**Comment by viniciusdeliz** [comment]
> Thanks @viniciusdeliz, I can see there are a few missing commits to apply to the translation here, could you take a look and update accordingly? https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/typescript.mdx?since=2023-10-23T20:48:38.000Z

@Yan-Thomas I've just ingested the latest changes and fixed some inconsistencies in the Portuguese version.

File: src/content/docs/pt-br/guides/typescript.mdx [hunk 1]
```diff
@@ -277,17 +299,19 @@ Para ver erros de tipagem no seu editor, por favor certifique-se de que você te
 `astro check` verifica todos os arquivos incluídos em seu projeto TypeScript. Para verificar tipos em arquivos Svelte e Vue, você pode usar os pacotes [`svelte-check`](https://www.npmjs.com/package/svelte-check) e [`vue-tsc`](https://www.npmjs.com/package/vue-tsc), respectivamente.
 :::
 
-📚 Leia mais sobre [a importação de arquivos `.ts`](/pt-br/guides/imports/#typescript) no Astro.
+import ReadMore from '~/components/ReadMore.astro'
+
+<ReadMore>Leia mais sobre [a importação de arquivos `.ts`](/en/guides/imports/#typescript) no Astro.</ReadMore>
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/guides/typescript.mdx:304]:
```suggestion
<ReadMore>Leia mais sobre [a importação de arquivos `.ts`](/pt-br/guides/imports/#typescript) no Astro.</ReadMore>
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

File: src/content/docs/pt-br/guides/typescript.mdx (hunk 1 above)
Comment (viniciusdeliz) [reply src/content/docs/pt-br/guides/typescript.mdx]: Thank you for the fix!

**Pull request merged**